scio-core/src/main/scala/com/spotify/scio/testing/TestDataManager.scala

```diff
@@ -112,6 +113,8 @@ case class BigQueryIO(tableSpecOrQuery: String) extends TestIO[TableRow](tableSp
 case class DatastoreIO(projectId: String, query: Query = null, namespace: String = null)
   extends TestIO[Entity](s"$projectId\t$query\t$namespace")
 
+case class ProtobufIO[T](path: String)(implicit ev: T <:< Message) extends TestIO(path)
```

Comment:
You shouldn't need the `ev` here. `saveAsProtobufFile` already has that constaint.
Does `[T <: Message]` work?